### PR TITLE
Add GitHub issue templates and bot for auto label assignment

### PR DESCRIPTION
**Pull request by abeykoon**
## Purpose

This PR is to add following templates for issues 

1. Bug report
2. Feature request
3. Improvement request
4. Documentation issue/improvement
5. Task
6. Epic

Also it contains a github action to auto assign label to issues - separating them out to relevant connector.

### CODE REVIEW

**Review by indikasampath2000 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/01_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    attributes:
+      label: Connector Name
+      description: Choose the connector you are reporting the bug against
+      options:
+        - connectors/openapi (Open API Connector)
+        - module/aws-s3 (AWS S3)
+        - module/aws-sqs (AWS SQS)
+        - module/azure-ad (Azure Active Directory)
+        - module/azure-eventhub (Azure Event Hub)
+        - module/azure-servicebus (Azure Servicebus)
```
Comment (LakshanSS) [inline .github/ISSUE_TEMPLATE/01_bug_report.yml:22]: Suggestion:-
`Azure Servicebus` -> `Azure Service Bus`

File: .github/ISSUE_TEMPLATE/01_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        - module/azure-storageservice (Azure Blob & Storage Service)
```
Comment (LakshanSS) [inline .github/ISSUE_TEMPLATE/01_bug_report.yml:23]: Suggestion:-
`(Azure Blob & Storage Service)` -> (Azure Blob & File Storage Service)

File: .github/ISSUE_TEMPLATE/02_documentation_issue.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      label: Connector Name
+      description: Choose the connector you are reporting the documentation issue/improvement against
+      options:
+        - connectors/openapi (Open API Connector)
+        - module/aws-s3 (AWS S3)
+        - module/aws-sqs (AWS SQS)
+        - module/azure-ad (Azure Active Directory)
+        - module/azure-eventhub (Azure Event Hub)
+        - module/azure-servicebus (Azure Servicebus)
+        - module/azure-storageservice (Azure Blob & Storage Service)
```
Comment (LakshanSS) [inline .github/ISSUE_TEMPLATE/02_documentation_issue.yml:23]: Same suggestions as above

File: .github/workflows/add_connector_label.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+                var word = words[i]
+                if(word.startsWith('Name\n\nmodule/') | word.startsWith('Name\n\nconnectors/')) {
+                    var label = word.substring(6) 
+                    console.log(label)
+                    github.issues.addLabels({
+                    issue_number: context.issue.number,
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    labels: [label]
+                     })
```
Comment (LakshanSS) [inline .github/workflows/add_connector_label.yml:25]: We can fix formatting

**Pull request merged**